### PR TITLE
feat(#50): Add read/write concurrency test for `MonoTojo`

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
I've added test that checks `MonoTojo` for concurrent reads and writes. And everything works like a charm. It' means I was mistaken with #50.
Actual problem with `MnSticky` which is not a thread safe: #51

Closes: #50

### CODE REVIEW

**Comment by volodya-lombrozo** [comment]
@yegor256 Can you have look. please?

File: pom.xml [hunk 1]
```diff
@@ -87,6 +91,11 @@ SOFTWARE.
       <artifactId>snakeyaml</artifactId>
       <version>1.33</version>
     </dependency>
+    <dependency>
+      <groupId>org.cactoos</groupId>
+      <artifactId>cactoos</artifactId>
+      <version>0.55.0</version>
```
Comment (yegor256) [inline pom.xml:97]: @volodya-lombrozo `scope` should be `test`?

Comment (volodya-lombrozo) [reply pom.xml]: @yegor256 sure, done

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**